packages/hash/external-services/kratos/kratos.dev.yml

```diff
@@ -37,6 +37,19 @@ selfservice:
       after:
         password:
           hooks:
+            # TODO - figure out whether this blocks writing to the datastore
+            - hook: web_hook
+              config:
+                url: http://host.docker.internal:5001/kratos-after-registration
```

Comment:
I assume this is in the same league as the `api_key` if we were to parameterize this through env var? It would be nice to have the workspace API URI be configurable

Reply:
Yeah, the whole value for `hooks` can be set by the `SELFSERVICE_FLOWS_REGISTRATION_AFTER_PASSWORD_HOOKS` environment variable. So that would include the workspace API URI.